### ISSUE TITLE
Edit/Update a User Static UI 

### ISSUE THREAD

**Issue by rosibaj** [icgc-argo/roadmap#60]
- [x] When the admin clicks the pencil icon from the users table, an edit modal appears:
- [x] The email address is disabled, this cannot be edited
- [x] The "Save" button is disabled until the admin user makes an update to the formfield values. The button is enabled (purple) when they have made a change.
Zeplin: https://zpl.io/VKDWoOE
![Screen Shot 2019-06-18 at 11 37 57 AM](https://user-images.githubusercontent.com/18424138/59698358-894ecc80-91bd-11e9-9508-d1e56813005e.png)

**Comment by ciaranschutte** [comment]
@rosibaj can we move the toast success into a seperate ticket because it's blocked by #149 (https://app.zenhub.com/workspaces/argo-5c8c8637777e5f767e486217/issues/icgc-argo/argo-platform/149) and I want to move these older ones through quickly

**Comment by rosibaj** [comment]
@ciaranschutte sorry to have missed your message - yes i will move to another ticket!

**Comment by rosibaj** [comment]
Moved here: https://github.com/icgc-argo/argo-platform/issues/132

**Comment by rosibaj** [comment]
@ciaranschutte I am still seeing this issue: 
![image](https://user-images.githubusercontent.com/15824252/61974766-51555900-afb5-11e9-91ea-c6121e7412cb.png)

**Comment by rosibaj** [comment]
fixin in @joneubank PR, but also needto change to email

**Comment by joneubank** [comment]
https://github.com/icgc-argo/argo-platform/pull/425
